handbook/engineering/onboarding/index.md

```diff
@@ -0,0 +1,41 @@
+# Engineering onboarding
+
+Welcome! We're excited to have you join the team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Getting set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete [general onboarding](../../people-ops/onboarding/index.md#general-onboarding-checklist)
+- [Configure your GitHub notifications.](../github-notifications/index.md)
+- Join #dev-announce, #dev-chat, and your team's channel on Slack, as well as any other channels you find interesting. [Team chat documentation](../../communication/team_chat.md#engineering)
+- Set up your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/main/doc/dev/getting-started/index.md). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
+- [Add Sourcegraph as a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). Add another entry for our internal instance: `https://sourcegraph.sgdev.org/search?q=%s`. This also ensures you have access to our internal instance.
```

Comment:
```suggestion
- [Add Sourcegraph as a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). Add another entry for our internal instance: `https://k8s.sgdev.org/search?q=%s`. This also ensures you have access to our internal instance.
```

Reply:
What is the difference between these two?

Reply:
The internal instance also has access to the private Sourcegraph repositories.

Reply:
`k8s.` is a full Kubernetes deployment, `sourcegraph.` is a single image deployment.